FSW/src/util/externalConfig.cpp

```diff
@@ -0,0 +1,31 @@
+/* externalConfig.cpp DEFINES all global objects declared with the 'extern' keyword in config.hpp
+    The compiler will point all files to these definitions, so that only one instance of each object exists globally.
+    DO NOT DEFINE THESE OBJECTS ANYWHERE ELSE!!!
+*/
+
+// Config Header
+#include "../headers/config.hpp"
+
+/* GLOBAL DEFINITIONS */
+// Main
+Event exitMainLoopEvent = Event();
+
+// Data Collection
+volatile bool STREAM_PHOTO_DATA = STREAM_PHOTO_DATA_INIT;
+RecurringEvent dataProcessEvent = RecurringEvent(SAMPLE_PERIOD_MSEC);
+Event saveBufferEvent = Event();
+TimedEvent sunriseTimerEvent = TimedEvent(WINDOW_LENGTH_MSEC);
+TimedEvent sweepTimeoutEvent = TimedEvent(SWEEP_TIMEOUT_MSEC);
+AsyncEvent downlinkEvent = AsyncEvent(MAXFILES);
+AsyncEvent scrubEvent = AsyncEvent(MAXFILES);
```

Comment:
They are equivalent, its just that `TimedEvent object(LENGTH)` is the same syntax as a function declaration, so
`TimedEvent object = TimedEvent(LENGTH)` leaves no ambiguity. 

Not to be confused with `TimedEvent object = new TimedEvent(LENGTH)`, which allocates memory on the heap instead of the stack.